lib/bulbs-elements/util/on-ready-or-now.test.js

```diff
@@ -0,0 +1,20 @@
+import onReadyOrNow from './on-ready-or-now';
+
+describe('onReadyOrNow', () => {
+  it('should callback now if document.readyState != "loading"', () => {
+    let spy = sinon.spy();
+    expect(document.readyState).to.not.eql('loading');
+    onReadyOrNow(spy);
+    expect(spy).to.have.been.called;
+  });
+
+  /* Again, I don't know how to mock out the document.readyState
```

Comment:
Could you create a `getDocumentReadyState` function in `on-ready-or-now` that the default function uses, then mock that out?